cypress.config.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  defaultCommandTimeout: 60000,
+  requestTimeout: 60000,
+  responseTimeout: 60000,
+  reporter: 'cypress-mochawesome-reporter',
```

Comment:
> Can the multi reports support the mochawesome you are adding here?

I saw the above comment from @cwperks too, and I'm having the same question. If I remember this correct, `cypress-multi-reporters` can be used to generate reports in multiple formats, including `Mochawesome`. But still, I know that `Mochawesome` should give us more detailed reports with information. So is there any specific reason for switching this to `Mochawesome`?

Reply:
We can check if that will work